### DISCUSSION TITLE
为《指北》的规则添加优先级

### DISCUSSION THREAD

**Discussion by gledos** [sparanoid/chinese-copywriting-guidelines#200] [General]
《指北》缺乏优先级关系，可能会存在一些冲突的情况，所以 gledos 认为如果《指北》的规则具有优先级，那么就能降低复杂程度，不再需要大量的例外，这样能够解决一部分潜在冲突。

比如「QQ音乐」是一个专有名词，下面是一些演示：

1.  这是一句含有QQ音乐的测试短句。
2.  这是一句含有 QQ音乐 的测试短句。
3.  这是一句含有 QQ 音乐的测试短句。
4.  这是一句含有 QQ音乐的测试短句。

上面的演示短句，应该如何书写才「正确」？下面是我为《指北》添加优先级，某种程度上能降低复杂程度，因为这个「工具」的原理也一并展示出来了。

---

## 空格

1.  「豆瓣FM」「QQ音乐」等产品名词、专有名词，按照官方所定义的格式书写。[^zy]
2.  度数／百分比与数字之间不需要增加空格。[^dw]
3.  全角标点与其他字符之间不加空格。
4.  全角数字被视作标点符号。
5.  全角与半角语言文字之间需要增加空格。

[^zy]: 如果不存在官方统一的写法，比如官网自相矛盾，那么也不一定需要遵守。

[^dw]: 如果有大量数字 + 单位的样式，建议参考专业的规范，比如美国国家标准与技术研究所的《SI 单位规则和样式约定清单》([_SI Unit rules and style conventions checklist_](https://physics.nist.gov/cuu/Units/checklist.html))。

示例：

> 单日 1 亿位用户打开 QQ音乐听音乐，同比增长 24.8%，用户总覆盖量更是达到 8 亿，跻身至移动 App 总排行榜 Top 15。

可以看到「QQ音乐」因为「空格.1」，不能使用空格来间隔，然后因为「空格.4」，需要在前方添加空格。

## 标点符号

1.  为文字假想一个语言环境，在不同的语言环境中使用不同的标点符号。
2.  如果片段环境的首尾有成对的书名号（包括斜体），那么成对的符号也应该使用片段环境语言的符号。

示例：

> apple 是传记 Steve Jobs 中 史蒂夫乔布斯 Steve Jobs 说过的词语

```text
         +======+======+==========+==+============+==========+===========+
    原文 | apple 是传记 Steve Jobs 中 史蒂夫乔布斯 Steve Jobs 说过的词语 |
         +======+======+==========+==+============+==========+===========+
语言环境 |      ·      ·          ·  ·  zh        ·          ·           |
         +------+------+----------+--+------------+----------+-----------+
片段环境 |  en  |      |    en    |  ·            |    en    |           |
         +------+------+----------+--+------------+----------+-----------+
类型判断 | 名词 |      |   书名   |  ·            |   人名   |           |
         +------+------+----------+--+------------+----------+-----------+
标点排版   (1)  ·      ·   (2)    ·  ，(3)        （   (4)   ）          。(3)
         +------+------+----------+--+------------+----------+-----------+
```

![圖片](https://github.com/sparanoid/chinese-copywriting-guidelines/assets/23720620/d9c89198-d0dc-499f-98af-aaaf9184f505)

> apple 是传记 _Steve Jobs_ 中，史蒂夫乔布斯（Steve Jobs）说过的词语。

1.  虽然英文在句首，但是此英文的片段环境没有占满一段文字，没有覆盖语言环境，所以不需要遵守英文文章首字母大写的规范，这里可以直接使用 apple。
2.  片段环境虽然无法影响整句，但会影响片段的首尾字符，比如书名号需要使用片段环境的，在这里是因为书籍，所以应该使用斜体的 _Steve Jobs_。
3.  这里整段都是中文环境，所以逗号、句号和括号使用中文的全角标点。

**Comment by liyishuai** [comment]
相关讨论：#110

“QQ音乐”的反例很精彩：
> 这是一句含有`[空格]`QQ`[空格]`音乐的测试短句。

作为“汉语文本”是不合理的，但——
> 这是一句含有`[间距]`QQ`[间距]`音乐的测试短句。

则是合理的“中文渲染”。

**Reply by gledos** [reply]
这个问题涉及《中文文案排版指北》的根源，即《指北》究竟属于「排版指导」或者「基于字符的排版指导」。

像 W3C Group 编写的 [中文排版需求](https://www.w3.org/TR/clreq/) 就属于「排版指导」，更全面和底层。

而在更广泛的网络平台上，比如 Twitter, Facebook 以及 Telegram 等地，无法使用专业排版工具，也没有 CSS 等样式的配置权限，就只能使用「基于字符的排版指导」。

gledos 认为《中文文案排版指北》属于解决在 Twitter, Facebook 以及 Telegram 等地发文的方案，所以不需要将空格理解为其他的内容。

但是也许可以编写一套合理的「排版指导」，然后在此基础上编写适用于 Twitter, Facebook 以及 Telegram 等地的「基于字符的排版指导」。

gledos 正在编写《[通用写作律法](https://github.com/saveweb/general-writing-laws)》，一种模块化的「基于字符的排版指导」，这样人人都能自定义出完善合理的「基于字符的排版指导」。（现在内容相当早期）
